pkg/sdn/api/validation/validation.go

```diff
@@ -40,7 +40,7 @@ func ValidateClusterNetworkUpdate(obj *sdnapi.ClusterNetwork, old *sdnapi.Cluste
 	if obj.Network != old.Network {
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("Network", obj.Network, "cannot change the cluster's network CIDR midflight."))
 	}
-	if obj.ServiceNetwork != old.ServiceNetwork {
+	if obj.ServiceNetwork != old.ServiceNetwork && old.ServiceNetwork != "" {
```

Comment:
I'm curious what updating an empty servicenetwork to a populated one does as far as propagating the change throughout the sdn